### PR TITLE
updated mobile skills + onboarding style

### PR DESCRIPTION
**Pull request by chrisdugne**
**Detailed purpose of the PR**

removed header, improved positionings for `app-review/skills/dashboard`

<p>
Screenshot 2023-01-23 at 15 49 52

Screenshot 2023-01-23 at 15 49 38
</p>

### CODE REVIEW

**Comment by chrisdugne** [comment]
[[OR] App - Dashboard révision - iteration UI](https://trello.com/c/rrhk41KY/2933-or-app-dashboard-r%C3%A9vision-iteration-ui)

**Comment by chrisdugne** [comment]
[Partage header / navbar entre les screens](https://trello.com/c/xatGraiS/2924-partage-header-navbar-entre-les-screens)

**Comment by chrisdugne** [comment]
[[OR] App - Dashboard Révision - Marge à réduire](https://trello.com/c/fs32Q3q0/2949-or-app-dashboard-r%C3%A9vision-marge-%C3%A0-r%C3%A9duire)

**Review by jomaora — APPROVED** [review]

File: packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx [hunk 1]
```diff
@@ -71,8 +76,8 @@ const createStyleSheet = (brandTheme: Brand, theme: Theme): StyleSheetType =>
     button: {
       height: 52,
       borderRadius: 7,
-      marginBottom: theme.spacing.large,
-      marginTop: theme.spacing.large,
+      marginBottom: 32,
```
Comment (scalabw) [inline packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx:79]: are we dropping the usage of theme ?

**Review by scalabw — APPROVED** [review]

File: packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx (hunk 1 above)
Comment (chrisdugne) [reply packages/@coorpacademy-components/src/atom/review-presentation/index.native.tsx:79]: https://coorpacademy.slack.com/archives/GA45KH3UM/p1674553275788439?thread_ts=1674488454.388169&cid=GA45KH3UM

**Pull request merged**